specification/mysql/resource-manager/readme.md

```diff
@@ -123,6 +123,15 @@ input-file:
 - Microsoft.DBforMySQL/preview/2020-07-01-preview/mysql.json
 ```
 
+### Tag: package-2021-05-01-preview
```

Comment:
similar to postresql, if you mean a separate SDK package, at least show it in tag name

Reply:
updated the tag name to package-flexibleserver-2021-05-01-preview